<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rfu</class>
 <widget class="QWidget" name="rfu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="lblOPRMODE_2">
   <property name="geometry">
    <rect>
     <x>796</x>
     <y>30</y>
     <width>291</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background:rgb(234, 236, 247);


color:rgb(53,74, 131);
</string>
   </property>
   <property name="text">
    <string>RFU</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>476</x>
     <y>80</y>
     <width>931</width>
     <height>149</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:color rgb(150,160,186,40%);
border-radius:10px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="lblHarnessLED">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>94</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHarness">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>BDOBUS Harness Check connection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckHarnessContinutyError">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>95</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_103">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>42</y>
      <width>61</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,1%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>DI's</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_104">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>44</y>
      <width>81</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,1%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>DO's</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_105">
    <property name="geometry">
     <rect>
      <x>582</x>
      <y>44</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,1%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>Continuty Error</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHarness">
    <property name="geometry">
     <rect>
      <x>196</x>
      <y>90</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border: 4px solid rgb(53,74, 131);
color:rgb(53,74, 131);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string>Harness</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_14" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>1621</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:color rgb(150,160,186,40%);
border-radius:10px;</string>
   </property>
   <widget class="QLabel" name="lblFireLED">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>160</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblBreechOpeningLED">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>236</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblBreechClosingLED">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>317</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckContinutyFire">
    <property name="geometry">
     <rect>
      <x>726</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckContinutyBreechOpening">
    <property name="geometry">
     <rect>
      <x>726</x>
      <y>236</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckContinutyBrechClosing">
    <property name="geometry">
     <rect>
      <x>726</x>
      <y>317</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckCrossContinutyFire">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckCrossContinutyBreechOpening">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>236</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckCrossContinutyBreechClosing">
    <property name="geometry">
     <rect>
      <x>905</x>
      <y>317</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_108">
    <property name="geometry">
     <rect>
      <x>579</x>
      <y>73</y>
      <width>81</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,1%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>DO's</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_109">
    <property name="geometry">
     <rect>
      <x>685</x>
      <y>70</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,0%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>Continuty
 Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_110">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>64</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,1%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>Cross Continuty
Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pbHarness_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>6</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Continuty</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFIre">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>155</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border: 4px solid rgb(53,74, 131);
color:rgb(53,74, 131);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string>FIRE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBreechOpening">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>232</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border: 4px solid rgb(53,74, 131);
color:rgb(53,74, 131);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string>BREECH OPENING </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblBreechClosing">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>312</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border: 4px solid rgb(53,74, 131);
color:rgb(53,74, 131);
border-radius:5px;



</string>
    </property>
    <property name="text">
     <string>BREECH CLOSING </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>630</y>
     <width>1821</width>
     <height>203</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:color rgb(150,160,186,40%);
border-radius:10px;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pbHarness_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Push Buttons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGRFD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>200</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>GRFD</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblBDOBUSJ17J21LED_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>82</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(234, 236, 247);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGRFD_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>200</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
color:white;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>GRFD</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckCrossBDOBUSJ17J21_3">
    <property name="geometry">
     <rect>
      <x>387</x>
      <y>105</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 18px; /* Adjust spacing between the checkbox and text */
    font-size: 28px; /* Set the font size */
    background-color: rgba(150, 160, 186, 1%); /* Background color */
 color:rgb(53,74, 131);
}

QCheckBox::indicator {
    width: 30px; /* Adjust the width of the checkbox indicator */
    height: 30px; /* Adjust the height of the checkbox indicator */
    border-radius: 30px; /* Make the indicator circular */
    border: 2px solid #cccccc; /* Add a border around the indicator */
color:rgb(53,74, 131);
}

QCheckBox::indicator:checked {
    background-color: #000000; /* Change the background color of the indicator when checked */
    border: 2px solid #ffffff; /* Add a border around the indicator when checked */
color:rgb(53,74, 131);
}

QCheckBox::indicator:unchecked {
    background-color: #ffffff; /* Change the background color of the indicator when unchecked */
color:rgb(53,74, 131);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_117">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>35</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,1%);
color:rgb(53,74, 131);</string>
    </property>
    <property name="text">
     <string>Cross Continuity
Error</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
